voter-dapp/src/ActiveRequests.js

```diff
@@ -603,6 +623,16 @@ function ActiveRequests({ votingAccount, votingGateway }) {
           </DialogContentText>
         </DialogContent>
       </Dialog>
+      {Object.keys(REQUEST_BLACKLIST).length > 0 && (
```

Comment:
Let's say that there is a price request that's been blacklisted in the past, but it's the request is longer active (for whatever reason). Would this toggle still show despite the fact that it won't do anything?

Not sure if this is a case that matters. It may be that these requests will always stay active after being blacklisted, so if there are any blacklisted, they will be in this table.

Reply:
> Would this toggle still show despite the fact that it won't do anything?

yes this toggle would show. Perhaps I should check that there are no current pending requests on the blacklist

Reply:
@mrice32 I changed the `Object.keys(REQUEST_BLACKLIST).length > 0` condition to: 

```
      {allPendingRequests.length >= pendingRequests.length && (<Component/>)
```

This will show the switch IFF there are any requests from `allPendingRequests` that have been filtered out.

Reply:
Nice and simple -- I like it